mev-relay-rs/src/relay.rs

```diff
@@ -448,6 +451,28 @@ impl Relay {
         state.auctions.insert(auction_request, auction_context);
         Ok(())
     }
+
+    fn verify_blinded_block_signature(
+        &self,
+        signed_block: &mut SignedBlindedBeaconBlock,
+    ) -> Result<(), ethereum_consensus::Error> {
+        let proposer = self
+            .validator_registry
+            .get_public_key(signed_block.message().proposer_index())
+            .unwrap();
+        let slot = signed_block.message().slot(); // signed_block.message().slot() + 1 ???;
```

Comment:
proposers are signing over the slot of the proposal, see here: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/validator.md#signature

Reply:
Do you have a preference between
`let slot = signed_block.message().slot();`  and
`let slot = &auction_request.slot` ?

Reply:
The 2nd's shorter, but the 1st is used elsewhere in the codebase

Reply:
I'd just get it from the `auction_request`